### PR TITLE
Add Dart to dependency scope table

### PR DESCRIPTION
**Pull request by AAugustine**
### Why:

Closes https://github.com/github/docs-content/issues/8691
Closes https://github.com/github/dependabot-alerts/issues/851

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Table located at: `data/reusables/dependabot/dependabot-alerts-dependency-scope.md` is being changed to include Dart

Screen Shot 2022-11-14 at 4 45 05 PM

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @AAugustine Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by mchammer01 — DISMISSED** [review]
@AAugustine 👋🏻 - this LGTM ✨ 
Inquired with you privately about versioning. Happy to merge this once we've established that there are no versioning considerations that we need to add 😃

File: data/reusables/dependabot/dependabot-alerts-dependency-scope.md
Comment (mchammer01) [inline data/reusables/dependabot/dependabot-alerts-dependency-scope.md:4]:
```suggestion
|:---|:---:|:---:|:---|{% ifversion fpt or ghec or ghes > 3.7 %}
```

File: data/reusables/dependabot/dependabot-alerts-dependency-scope.md
Comment (mchammer01) [inline data/reusables/dependabot/dependabot-alerts-dependency-scope.md:6]:
```suggestion
| Dart | pub | pubspec.lock |  ✔ |{% endif %}
```

**Review by mchammer01 — COMMENTED** [review]
Adding the relevant versioning to the two new rows.
Will test this on staging before merging.

**Review by mchammer01 — APPROVED** [review]
LGTM 💖 
Thanks for double-checking the versioning for me 🙇🏻

**Pull request merged**
